rsr_convert_scripts/ghi_rsr.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018-2022 Pytroll developers
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Read the FY-4B GHI relative spectral responses.
+
+Data from http://satellite.nsmc.org.cn/PortalSite/StaticContent/FileDownload.aspx?CategoryID=1&LinkID=687
```

Comment:
The link doesn't seem to work any more. This lets me download it though:
`http://satellite.nsmc.org.cn/PortalSite/StaticContent/FileDownload.aspx?currentculture=zh-CN&CategoryID=1&LinkID=687`

Reply:
Well, looks like they just changed http to https. I thought it was a new version at first look....